packages/@aws-cdk/aws-kinesisfirehose-alpha/lib/delivery-stream.ts

```diff
@@ -216,16 +217,16 @@ export interface DeliveryStreamProps {
   /**
    * Indicates the type of customer master key (CMK) to use for server-side encryption, if any.
    *
-   * @default StreamEncryption.UNENCRYPTED - unless `encryptionKey` is provided, in which case this will be implicitly set to `StreamEncryption.CUSTOMER_MANAGED`
+   * @default StreamEncryption.unencrypted()
    */
   readonly encryption?: StreamEncryption;
 
-  /**
-   * Customer managed key to server-side encrypt data in the stream.
-   *
-   * @default - no KMS key will be used; if `encryption` is set to `CUSTOMER_MANAGED`, a KMS key will be created for you
-   */
-  readonly encryptionKey?: kms.IKey;
+  // /**
+  //  * Customer managed key to server-side encrypt data in the stream.
+  //  *
+  //  * @default - no KMS key will be used; if `encryption` is set to `CUSTOMER_MANAGED`, a KMS key will be created for you
+  //  */
+  // readonly encryptionKey?: kms.IKey;
```

Comment:
Is this intended (commented out)?

Reply:
Forgot to remove that, will do that now.